pkg/spdk/backing_image.go

```
@@ -0,0 +1,703 @@
+	}
+	exposedSnapshotLvolAddress, err := srcBackingImageServiceCli.BackingImageExpose(bi.ctx, bi.Name, srcLvsUUID)
+	if err != nil {
+		return errors.Wrapf(err, "failed to expose the source backing image %v", bi.Name)
+	}
+	externalSnapshotLvolName := GetBackingImageSnapLvolName(bi.Name, srcLvsUUID)
+
+	srcIP, srcPort, err := splitHostPort(exposedSnapshotLvolAddress)
+	if err != nil {
+		return errors.Wrapf(err, "failed to split host and port from address %v", exposedSnapshotLvolAddress)
+	}
+	_, _, err = connectNVMeTarget(srcIP, srcPort)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)